developer-docs/how-to-guide/configure_content_metafilter.md

```diff
@@ -0,0 +1,197 @@
+      <td>Variable is used to blacklist the framework whose content should not be displayed</td>
+      <td><blank></td>
+      <td>Content Service</td>
+    </tr>
+     <tr>
+      <td>5</td>
+      <td>sunbird_content_filter_contenttype_whitelist</td>
+      <td>Configures the contentType whose content can be displayed in the portal. This is a comma-separated string ex:”A,B,C”where A,B,C are different contentType</td>
+      <td>Variable is used to whitelist the contentType whose content should be displayed</td>
+      <td><blank></td>
+      <td>Content Service</td>
+    </tr>
+     <tr>
+      <td>6</td>
+      <td>sunbird_content_filter_contenttype_blacklist</td>
+      <td>Configures the contentType whose content should not be displayed in the portal this is a comma-separated string ex:”X,Y,Z”where X,Y,Z are contentType</td>
+      <td>Variable is used to blacklist the contentType whose content should be displayed</td>
+      <td><blank></td>
+      <td>Content Service</td>
+    </tr>
+    <tr>
+      <td>7</td>
+      <td>sunbird_content_filter_resourcetype_whitelist</td>
+      <td>Configures the resourceType whose content can be displayed in the portal. This is a comma-separated string ex:”A,B,C”where A,B,C are different resourceType</td>
+      <td>Variable is used to whitelist the resourceType whose content should be displayed</td>
+      <td><blank></td>
+      <td>Content Service</td>
+    </tr>
+    <tr>
+      <td>8</td>
+      <td>sunbird_content_filter_resourcetype_blacklist</td>
+      <td>Configures the resourceType whose content should not be displayed in the portal this is a comma-separated string ex:”X,Y,Z”where X,Y,Z are resourceType</td>
+      <td>Variable is used to blacklist the resourceType whose content should be displayed</td>
+      <td><blank></td>
+      <td>Content Service</td>
+    </tr>
+     <tr>
+      <td>9</td>
+      <td>sunbird_content_filter_mimetype_whitelist</td>
+      <td>Configures the mimeType whose content can be displayed in the portal. This is a comma-separated string ex:”A,B,C”where A,B,C are different mimeType</td>
+      <td>Variable is used to whitelist the mimeType whose content should be displayed</td>
+      <td><blank></td>
+      <td>Content Service</td>
+    </tr>
+     <tr>
+      <td>10</td>
+      <td>sunbird_content_filter_mimetype_blacklist</td>
+      <td>Configures the mimeType whose content should not be displayed in the portal this is a comma-separated string ex:”X,Y,Z”where X,Y,Z are mimeType</td>
+      <td>Variable is used to blacklist the mimeType whose content should not be displayed</td>
+      <td><blank></td>
+      <td>Content Service</td>
+    </tr>
+  </tbody>
+  </table>
+
+## Concepts Covered
+
+<This section should cover in brief, the concepts covered.>
+
+## Related Topics
+
+<This section should provide links to related topics.>
```

Comment:
This file does not follow the structure of a 'How to' doc. Nor are all the sections complete. As a result, this PR cannot be merged. Please do not send PRs for new files along with updated ones. it stalls the merge and publish process.